open-hackathon-server/src/hackathon/hack/team_manager.py

```diff
@@ -788,3 +790,42 @@ def __get_hackathon_and_show_detail(self, team):
         team_dic = team.dic()
         team_dic["hackathon"] = hack_manager.get_hackathon_detail(team.hackathon)
         return team_dic
+
+    def __email_notify_dev_plan_submitted(self, team):
+        # send emails to all admins of this hackathon when one team dev plan is submitted.
+        admins = UserHackathon.objects(hackathon=team.hackathon, role=HACK_USER_TYPE.ADMIN).distinct("user")
```

Comment:
Please send to me an email as a test. I want to check its effectiveness and appearance.